Resources/Private/Fusion/OpenCookieManagerButton.fusion

```diff
@@ -0,0 +1,27 @@
+prototype(Sandstorm.CookieConsent:OpenCookieManagerButton) < prototype(Neos.Neos:ContentComponent) {
+    renderer = Sandstorm.CookieConsent:Component.OpenCookieManagerButton {
+        buttonText = Neos.Neos:Editable {
+            property = "text"
+            block = false
+        }
+
+        onClick = "(function showCookieSettings(e) { e.preventDefault(); klaro.show(); })(event)"
```

Comment:
@leon note: I changed the clickhandler function and added an href property to the `<a>` on line 22. This is so that the link has a cursor by default (I didn't want to change the css for this). To make sure that the default behavior is not triggered (page reload), i also added a `preventDefault()`-call.